lib/js/_publish/_add-files-to-package-json.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * _add-files-to-package-json.js
+ */
+'use strict';
+
+const fs = require('fs');
+
+function addFilesToPackageJSON(path) {
+	return new Promise((resolve, reject) => {
+		try {
+			fs.readFile(`${path}/package.json`, (err, data) => {
+				const json = JSON.parse(data);
+				json.files = ['scss', 'js', 'view', 'img'];
```

Comment:
this shouldn't be hard-coded in here, it is too brittle, and there may well be more files that need to be included depending on the context this is being used in